datasets/in/mha_banned/crawler.py

```diff
@@ -65,11 +81,6 @@ def crawl(context: Context) -> None:
         for row in csv.DictReader(fh):
             entity = context.make(row.pop("Type", "LegalEntity"))
             source_url = row.pop("SourceURL")
-            if source_url not in expected_sources:
-                context.log.warn(
-                    "Source URL not in overview page. Perhaps it's out of date?",
-                    url=source_url,
-                )
```

Comment:
why are we removing this? This is our sanity check that all our sources are still linked from the landing page. We don't want to give this up.